node-16.0/README.md

```diff
@@ -0,0 +1,46 @@
+# 📧 Your Function Name
+
+Brief description:
+
+<!-- For example 'Create Stripe payment and return payment URL' -->
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud functions:
+
+- **APPWRITE_FUNCTION_ENDPOINT** - Endpoint of Appwrite project
+- **APPWRITE_FUNCTION_API_KEY** - Appwrite API Key
+<!-- Add your custom environments variables -->
+
+## 🤖 Documentation
+
+_Example input:_
+
+No input
+
+<!-- If input is expected, add example -->
+
+_Example output:_
+
+<!-- Update with your expected output -->
+
+```json
+{
+    "areDevelopersAwesome": true
+}
+```
+
+## 🚀 Deployment
+
+1. Install [Appwrite CLI](todo-link-to-cli)
+2. Authenticate with `appwrite login`
+3. Open folder with your `appwrite.json`
+4. Start deploy process with `appwrite deploy`
```

Comment:
These steps will change a bit. 

Remember, this repo will server two purposes. 
1) Template that the CLI will clone
2) Template that developers can use irrespective of the CLI.

So these deployment instructions ( if we plan to keep this section ) should apply to both audiences.

Reply:
This would get too long if we wanted to mention both possible ways and would be duplicated over every single function. Let's remove it since the root folder of the project should have a deployment section (depending on if created with CLI or any other way).